.gitignore

```diff
@@ -29,3 +29,9 @@ sphinx
 *.ist
 wp.pdf
 scripts/_tmp
+catboost_info/catboost_training.json
+catboost_info/learn_error.tsv
+catboost_info/test_error.tsv
```

Comment:
isn't it better to ignore all catboost folder? Moreover ,it should be deleted with the previous implementation

Reply:
Well it don't get deleted after I launch integration tests.